.stylelintrc

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": [
+    "stylelint-config-twbs-bootstrap/scss"
+  ],
+  "rules": {
+    "color-named": null,
```

Comment:
@jakearchibald Do you prefer having this off? I fixed white and black since they are easy ones, but there are 3 more instances of named colors so I thought I'd leave this off.

Reply:
Agreed. It seems like a weird rule to me.